webpack.config.js

```diff
@@ -1,3 +1,56 @@
+const path = require('path');
+const {
+  getWebpackDevServerConfig,
+  getWebpackBundleConfig,
+} = require('codemirror-blocks/lib/toolkit/webpack');
+
+const devServerConfig = getWebpackDevServerConfig({
+  context: path.resolve(__dirname, 'dev-server'),
+  entry: './index.js',
+});
+const bundleConfig = getWebpackBundleConfig({
+  entry: {
+    CodeMirrorBlocks: path.resolve(__dirname, 'src', 'languages', 'wescheme', 'index'),
+  },
+});
+
+const alias = {
+  jsnums: 'wescheme-js/src/runtime/js-numbers',
+  lex: 'wescheme-js/src/lex',
+  types: 'wescheme-js/src/runtime/types',
+  structs: 'wescheme-js/src/structures',
+};
+
+devServerConfig.module.rules.push({
+  test: /\.rkt$/,
+  use: [{ loader: 'raw-loader' }],
+});
+
+// Add aliases needed by WeschemeParser.js
+// TODO(pcardune): stop using aliases and just import
+// directly from the right place...?
+devServerConfig.resolve.alias = alias;
+bundleConfig.resolve.alias = alias;
+
+bundleConfig.externals = {
+  'codemirror': 'codemirror',
+  'codemirror/addon/search/search' : 'codemirror',
+  'codemirror/addon/search/searchcursor' : 'codemirror',
+  'jsnums': 'jsnums',
+  'lex': {
+    commonjs: 'plt.compiler',
+    commonjs2: 'plt.compiler',
+    root: ['plt','compiler'],
+  },
+  'types': 'types',
+  'structs': {
+    commonjs: 'plt.compiler',
+    commonjs2: 'plt.compiler',
+    root: ['plt','compiler'],
+  },
```

Comment:
This configuration is a great example of why bundling should happen as part of the build process for wescheme.org, and not in this repo. This configuration is super-duper specific to the way wescheme.org works, and the resulting bundle would be unusable in any other context.

Reply:
1. The spec/ui/*-test-old.js files can be safely removed from this repo. They _should_, however, be resurrected in the CMB core repo. When we switched Drag-n-Drop libraries a while back, a lot of these tests broke.
2. See my comment in CMB re: generating a separate CSS file
3. You're right that this configuration is crazy-specific to wescheme. But then again, (a) the WeScheme repo itself has a build system designed in 2008 and isn't getting that love anytime soon and (b) this language repo is specific to WeScheme as a use-case anyway. Do you think this is sufficiently offensive that it's worth overhauling the build process for another repo altogether?

Reply:
1. ok, I created a separate issue for test files.
2. ok, I'm just going to leave the css files bundles with js.
3. I don't think you have to overhaul anything. I think you could just delete this repo altogether and make it a subdirectory of the wescheme repo. That would make it more clear that this was application code and not some kind of reusable library.

Reply:
That is an _excellent_ idea. I'll add it as an issue.